Deploy the new version of the website (#1086)
pyne/pyne: .circleci/README.md

PyNE-CI Documentation
=====================

> PyNE Circle-CI use CIrcle_CI version 2.1 and is composed in 4 main sections:

Reviewer: ```suggestion
PyNE Circle-CI use CIrcle_CI version 2.1 and is composed of 4 main sections:
```

`executors`, `commands`, `jobs` and `workflow`.

Executors
---------

> The `executors` define aliases to the environment to run the different jobs, in our case

Reviewer: ```suggestion
The `executors` define aliases to the environment to run the different jobs. In our case these are
```

> The `executors` defines aliases to the environment to run the differents jobs, in our case

Reviewer 2: ```suggestion
The `executors` define aliases to the environment to run the different jobs, in our case
```

> different docker images containing differents combinaisons of the optionnal

Reviewer 2: ```suggestion
different docker images containing different combinations of the optional
```

dependencies of PyNE (`MOAB`, `pyMOAB`, `DAGMC`) in one of the two supported

> PyTHON flavors (Python 2.7 or Python 3.6).

Reviewer: ```suggestion
Python versions (Python 2.7 or Python 3.6).
```

> PyTHON flavor (Python 2.7 or Python 3.6).

Reviewer 2: ```suggestion
PyTHON flavors (Python 2.7 or Python 3.6).
```

Commands
---------

> The `commands` section allows to define macro commands with arguments to be used

Reviewer 2: ```suggestion
The `commands` section allows the definition of macro commands with arguments to be used
```

> The `commands` section allows the definition of macro commands with arguments to be used

Reviewer: ```suggestion
The `commands` section provides the definitions of macro commands with arguments to be used
```

Reviewer: I think this wording is more of what you wanted here? Not sure "allows" is truly what is happening.

> in the differents jobs.

Reviewer 2: ```suggestion
in the different jobs.
```

> `news_check`: check the presence of a new news file in the news folder

Reviewer: ```suggestion
`news_check`: checks the presence of a new news file in the news folder
```

> `save_container`: allows to save the status of a container to be reloaded in an

Reviewer 2: ```suggestion
`save_container`: saves the status of a container to be reloaded in an
```

other job.

    - `arguments`: 
        - `build` (string): build configuration parameters, used to identify a
          build configuration

`pull_container`: pulls a previously saved (using the `save_container` command) status of container.

    - `arguments`:
        - `build` (string): build configuration parameters (has to match the build
          argument used to save the container)

> `checkout_build`: checkout PyNE branch to be tested, build its and saves it

Reviewer 2: ```suggestion
`checkout_build`: checks out PyNE branch to be tested, builds it and saves it
```

> usins the `save_container` command.

Reviewer 2: ```suggestion
using the `save_container` command.
```

    
    - `arguments`:
        - `build` (string): used to identify a configuration
        - `flags` (string): flags to be use when building PyNE

> `run_test`: pull a previous using `pull_container` command, then run the PyNE

Reviewer: ```suggestion
`run_test`: pulls a previous container using `pull_container` command, then runs the PyNE
```

nosetests using the provided `flag`.
    
    - `arguments`:
        - `build` (string): used to identify a configuration
        - `flags` (string): flags to be use when running the PyNE nosetests

> `website_build_push`: pull the Python 2.7 build with all the PyNE optionnal

Reviewer: ```suggestion
`website_build_push`: pull the Python 2.7 build with all the PyNE optional
```

> depedencies `python2_dagmc_pymaob` saved image (using `pull_container` command)

Reviewer: ```suggestion
depedencies `python2_dagmc_pymoab` saved image (using `pull_container` command),
```

> build the website and push it to the `pyne.github.com` repo.

Reviewer: ```suggestion
build the website, and push it to the `pyne.github.com` repo.
```

    
    - `arguments`:

>         - `push_option` (string: `test` or `root`): 

Reviewer: I am very confused based on this specific part of the documentation what the options are? Are they `test`, `root`, and `master`??

Author: this part is messed up... 
sorry about that....

>             - `test` option will push the new built website to `website_preview` 

Reviewer: ```suggestion
            - `test` option will push the newly built website to `website_preview` 
```

>             branch of the repo allowing previous allowing reviews. the push `root` 

Reviewer: ```suggestion
            branch of the repo allowing reviews. The push `root` 
```

            version will push the website on the

>             - `master` branch of the repo, deploying a new version of the website 

Reviewer: confused by formatting here - why does this start at a new bullet?

            in `pyne.io`

Jobs
----
The `jobs` section defines all the different jobs used in the different
workflows. For each built configuration two jobs have been defined, one to build

> PyNE, and one to run the `nosetests`. In addition to the the build and test

Reviewer: ```suggestion
PyNE, and one to run the `nosetests`. In addition to the build and test
```

jobs, two additional jobs have been created to build and push the website, one
to the `website_test` branch of the `pyne.github.com` repo, and one to the
`master` branch.

Workflow
--------

> The `workflow` section defines independant workflow triggers the differents

Reviewer: ```suggestion
The `workflow` section defines independent workflow triggers the different
```
